ext/drcovlib/drcovlib.c

```diff
@@ -213,7 +213,7 @@ static void
 dump_drcov_data(void *drcontext, per_thread_t *data)
 {
     version_print(data->log);
-    module_table_print(module_table, data->log, false);
+    module_table_print(module_table, data->log, options.verbose_module_table);
```

Comment:
You mean, if I run drcov -verbose today, it will fail to post-process?  It seems like it might be best to support that in general, right, with extra info simply supplying extra diagnostic info and not breaking functionality?